src/OpenConext/ProfileBundle/Service/GlobalViewParameters.php

```diff
@@ -19,74 +21,45 @@
 namespace OpenConext\ProfileBundle\Service;
 
 use OpenConext\Profile\Assert;
-use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface ;
 
 final class GlobalViewParameters
 {
-    /**
-     * @var TranslatorInterface
-     */
-    private $translator;
-
-    /**
-     * @var array
-     */
-    private $helpUrls;
-
-    /**
-     * @var array
-     */
-    private $privacyUrls;
+    private array $helpUrls;
 
-    /**
-     * @var array
-     */
-    private $platformUrls;
+    private array $privacyUrls;
 
-    /**
-     * @var array
-     */
-    private $termsOfServiceUrls;
+    private array $platformUrls;
 
-    /**
-     * @var array
-     */
-    private $profileExplanationImageUrls;
+    private array $termsOfServiceUrls;
 
-    /**
-     * @var array
-     */
-    private $attributeInformationUrls;
+    private array $profileExplanationImageUrls;
 
-    /** @var bool */
-    private $removeConsentEnabled;
+    private array $attributeInformationUrls;
 
     public function __construct(
-        TranslatorInterface $translator,
+        private readonly TranslatorInterface $translator,
         array $locales,
         array $helpUrls,
         array $privacyUrls,
         array $platformUrls,
         array $termsOfServiceUrls,
         array $profileExplanationImageUrls,
         array $attributeInformationUrls,
-        bool $removeConsentEnabled
+        private readonly bool $removeConsentEnabled,
```

Comment:
Maybe first list the promoted parameters, followed by the regular ones. But then, the callers need to update their call.. 

/nitpick

Reply:
There's no aggreement over this I believe. But, if the calles use parameter naming, the call would have to be rewritten.

Leaving it for now

Reply:
Surething boss!